org.testeditor.web.backend.xtext/build.gradle

```diff
@@ -42,3 +42,18 @@ dependencyCheck {
     failBuildOnCVSS=11
     suppressionFile=file("suppressions.xml").getCanonicalPath()
 }
+
+task docker(type:Exec) {
+    dependsOn(build)
+    executable 'sh'
+    args '-c', """
+      (([ -d testeditor ] && rm -Rf testeditor) || true) &&
+      cd build/distributions && (rm xtext.tar || true) &&
+      rm org.testeditor.*.zip &&
+      mv *.tar xtext.tar &&
```

Comment:
same as above, renaming in this manner might break when there happens to be an additional, random tar file in the directory.